rest/model/conversions.go

```diff
@@ -17,56 +17,77 @@ type typeInfo struct {
 	Type string
 }
 
+const (
+	buildFromService = "BuildFromService"
+	toService        = "ToService"
+)
+
 // conversionFn uses the specific naming conventions of type conversion funcs in conversions.gotmpl
-// to find the correct generated type conversion function
-func conversionFn(in types.Type, outIsPtr bool, generatedConversions map[string]string) (string, error) {
+// to find the correct generated type conversion function. It returns the name of the conversion function,
+// then the name of the inverse conversion function
+func conversionFn(in types.Type, outIsPtr bool, generatedConversions map[string]string) (string, string, error) {
```

Comment:
Would it be worth using a struct Names with Converter/Inverter to make sure this is clear in all the helper functions? It can be easy to mix up strings.